<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>751</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Nova Cond</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des athlètes</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>731</width>
       <height>411</height>
      </rect>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prénom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Sexe:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Ajouter">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:blue;
color:black;
border-style:outset;
border-width: 2px;
border-radius: 7px</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>150</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Nom_ajout">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nationnalité:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom_ajout">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_sexe_ajout">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>100</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Masculin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Feminin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Autre</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nation_ajout">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_age_ajout">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>150</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ID_ajout">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>200</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Spécialité:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_specielite_ajout">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prénom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nationalité:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Sexe:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_modifier">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_modifier">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom_modifier">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ntionalite_mofier">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:blue;
color:black;
border-style:outset;
border-width: 2px;
border-radius: 7px</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Spécialité:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_specialite_modifier">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_sexe_modifier">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
       <item>
        <property name="text">
         <string>Masculin </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Feminin </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>autres</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_age_modifier">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>110</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Supprimer</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:blue;
color:black;
border-style:outset;
border-width: 2px;
border-radius: 7px</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>351</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nationalité</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>age</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>130</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_recherche">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>130</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;
border-style:outset;
border-width: 2px;
border-radius: 10px</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
